### DISCUSSION TITLE
[백준] 18870 좌표 압축 | Jae’s Blog

### DISCUSSION THREAD

**Discussion by giscus** [jaenam615/jaenam615.github.io#11] [Announcements]
# [백준] 18870 좌표 압축 | Jae’s Blog

백준 2156 난이도: 실버 2 알고리즘: 정렬

https://jaenam615.github.io/coding/baekjoon/2024-02-23-boj18870/

**Comment by EririnG** [comment]
많이 배우고 갑니다. 이런 방식도 있는지 처음 알았네요
